Dockerfile

```diff
@@ -30,10 +30,11 @@ RUN cd /opt && tar -xvzf android-sdk.tgz
 RUN cd /opt && rm -f android-sdk.tgz
 
 # Grab what's needed in the SDK
+RUN echo y | android update sdk --no-ui --all --filter tools > /dev/null 
```

Comment:
can you explain why this different pattern to install `tools`? Maybe a comment will help not changing it to be like the others by mistake.